tests/unit/s2n_server_certificate_status_test.c

```diff
@@ -158,7 +158,15 @@ int main(int argc, char **argv)
         EXPECT_SUCCESS(s2n_stuffer_growable_alloc(&stuffer, 0));
 
         EXPECT_SUCCESS(s2n_tls13_server_status_request_extension.send(conn, &stuffer));
-        EXPECT_FAILURE_WITH_ERRNO(s2n_tls13_server_status_request_extension.recv(conn, &stuffer), S2N_ERR_CERT_UNTRUSTED);
+
+#if defined(OPENSSL_IS_BORINGSSL)
+        /* s2n_x509_validator_validate_cert_stapled_ocsp_response returns untrusted error if ocsp is not supported */
+        EXPECT_FAILURE_WITH_ERRNO(s2n_tls13_server_status_request_extension.recv(conn, &stuffer),
+                S2N_ERR_CERT_UNTRUSTED);
+#else
+        EXPECT_FAILURE_WITH_ERRNO(s2n_tls13_server_status_request_extension.recv(conn, &stuffer),
+                S2N_ERR_INVALID_OCSP_RESPONSE);
+#endif
```

Comment:
We generally try to only use conditional compilation if we can't do a runtime check, like if a function we want to call might not exist. For this case, can you just do a runtime if/else with [s2n_libcrypto_is_boringssl](https://github.com/aws/s2n-tls/blob/main/crypto/s2n_openssl.h#L56)?

Reply:
Right okay. I fixed this in 0cc0d4b398067aeb65e95af4b5d4d80de7f2a0ed and also used `s2n_x509_ocsp_stapling_supported` instead to standardize the use of `S2N_OCSP_STAPLING_SUPPORTED` rather than just `OPENSSL_IS_BORINGSSL`.